tests/explainers/test_linear.py

```diff
@@ -7,6 +7,13 @@
 import pytest
 import shap
 
+# Ignore expected internal shap warnings about deprecated syntax in LinearExplainer
+# In future when the deprecated syntax is fully removed, the tests must be updated
+pytestmark = [
```

Comment:
I don't love this, but I suppose we can remove the deprecated syntax after the first release.Perhaps we should raise an issue to make sure we follow through on fully removing deprecated syntax and updating this test.

Reply:
Yes, agreed. That's probably not something for the very first release; but, once we're in the swing of regular releases, it will be a nice thing to deprecate and remove.